runtime/pkg/fileutil/fileutil.go

```diff
@@ -125,3 +126,19 @@ func ForceRemoveFiles(paths []string) {
 		_ = os.Remove(path)
 	}
 }
+
+func ExpandHome(path string) (string, error) {
+	if path == "" || path[0] != '~' {
+		return path, nil
+	}
```

Comment:
If `len(path) > 1`, then it must also check that `path[1] == '/'`. Because `~filename.txt` is a valid regular filename.

Reply:
Ah got it. Was wondering what this use case is for.